Use_Cases/Opportunity_List/README.md

```diff
@@ -0,0 +1,28 @@
+# Opportunity List
+
+Easily view opportunities in a single screen both in chart and table format
+
+<img  src="opportunity_list.png"  width="300"></img>
+
+The table shows all opportunities while the charts and KPIs offer an overview. Search or filter to drill down to the opportunities you want to see, or click on portions of a chart to drill down and see the opportunities in that section. In addition, create a new opportunity using a step-by-step process.
```

Comment:
I think this list would be clearer rendered as a list

```suggestion
The table shows all opportunities, while the charts and KPIs offer an overview. 

With this page you can do any of the following: 

- Search or filter to drill down to the opportunities you want to see
- Click on portions of a chart to drill down and see the opportunities in that section
- Create a new opportunity using a step-by-step process
```

Reply:
Okay, I'll do that. Thanks for the feedback!